roles/dns/README.md

```diff
@@ -0,0 +1,42 @@
+dns
+===
+
+Configure a DNS server serving IPs of all the nodes of the cluster
+
+Requirements
+------------
+
+None
+
+Role Variables
+--------------
+
+| Name | Mandatory / Optional | Description |
+|------|----------------------|-------------|
+| `dns_zones` | Mandatory | DNS zones in which we must find the hosts |
+| `dns_forwarders` | If not set, the DNS will be a recursive non-forwarding DNS server | DNS forwarders to delegate the requests for hosts outside of `dns_zones` |
+| `g_all_hosts` | Mandatory | Exhaustive list of hosts |
```

Comment:
Thanks for the review @twiest.

I’ve created a dedicated `dns_all_hosts` role variable.